packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -79,14 +82,11 @@ export abstract class ActorQueryOperation extends Actor<IActionQueryOperation, I
 
   public static getExpressionContext(context: ActionContext): IExpressionContext {
     if (context) {
-      const now: Date | undefined = context.get(KEY_CONTEXT_QUERY_TIMESTAMP, undefined);
```

Comment:
I would have left this explicit undefined in to get developer attention to the fact that the key can be not present, but it doesn't really matter.